plugins/com.google.cloud.tools.eclipse.test.util/META-INF/MANIFEST.MF

```diff
@@ -10,7 +10,8 @@ Bundle-ActivationPolicy: lazy
 Require-Bundle: com.google.cloud.tools.eclipse.test.dependencies,
  org.eclipse.ui.ide,
  org.eclipse.core.jobs,
- org.eclipse.wst.validation
+ org.eclipse.wst.validation,
+ org.eclipse.equinox.preferences;bundle-version="3.5.300"
```

Comment:
why bundle here instead of import?

Reply:
I had thought I would be bringing in classes the org.eclipse.core.runtime package, which is split.  But I'm not.